### PR TITLE
Completed Product Listing Page

### PR DESCRIPTION
**Pull request by varun1233**
Completed Product Listing Page

### CODE REVIEW

File: pages/products-listing/products.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  color: var(--primary-font-color);
+}
+.product-filters {
+  max-width: 20%;
+  width: 100%;
+  background: var(--primary-background-color);
+  padding: 1.5rem 2.5rem;
+  overflow-x: auto;
+  position: sticky;
+  top: 0rem;
```
Comment (Adarsh-Patel-Dev) [inline pages/products-listing/products.css]: You can use simply 0

File: pages/products-listing/products.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  padding: 3rem;
+  font-size: 1.8rem;
+  line-height: 1.5;
+}
+.product-list-row {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1rem;
+  justify-content: space-between;
+  box-sizing: border-box;
```
Comment (Adarsh-Patel-Dev) [inline pages/products-listing/products.css]: box-sizing: border-box;
try to use this property in reset.css

**Review by Mohitk391 — APPROVED** [review]
Appreciation: The code looks clean and you have used variables, correct units at respective places.
The UI looks awesome as well. Overall, you have done great work.
The branch is good to merge.

**Review by Adarsh-Patel-Dev — APPROVED** [review]
Suggestd some changes
Rest Code structure is clean. 
UI is awesome.

File: pages/products-listing/products.css (hunk 2 above)
Comment (gouravm20) [reply pages/products-listing/products.css]: *{box-sizing: border-box;} may be good option

**Review by gouravm20 — COMMENTED** [review]
look good , not  major issue ready to deploy

File: pages/products-listing/products.css [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  width: 1rem;
+  height: 1.5rem;
+  outline: none;
+  display: inline-block;
+  position: relative;
+  top: 0.2rem;
+}
+.product-list {
+  max-width: 80%;
+  width: 90%;
```
Comment (malaykhakhar) [inline pages/products-listing/products.css:71]: max-width less than the width?

File: pages/products-listing/products.css (hunk 1 above)
Comment (varun1233) [reply pages/products-listing/products.css]: Done

File: pages/products-listing/products.css (hunk 2 above)
Comment (varun1233) [reply pages/products-listing/products.css]: Done

**Review by malaykhakhar — APPROVED** [review]
The UI is very pretty. The code looks very good too. You can merge

**Pull request merged**